test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -481,6 +481,8 @@ Function sys.int4int2_sum(bigint)
 Function sys.int4rowversion(integer,integer,boolean)
 Function sys.int4varbinary(integer,integer,boolean)
 Function sys.int8_fixeddecimal_cmp(bigint,sys.fixeddecimal)
+Function sys.int8_to_money(bigint)
```

Comment:
can we add dependency test for these so no entries come here, talk to Rishabh

Reply:
Added